doc-content/kogito-docs/src/main/asciidoc/openshift/proc_kogito-travel-agency-enable-persistence.adoc

```diff
@@ -0,0 +1,32 @@
+[id='proc_kogito-travel-agency-enable-persistence']
+
+= Enabling Infinispan persistence for {PRODUCT} services
+
+{PRODUCT} supports runtime persistence for process data in your services. {PRODUCT} persistence is based on https://infinispan.org/[Infinispan] and enables you to configure key-value storage definitions to persist data, such as active nodes and process instance variables, so that the data is preserved across application restarts.
+
+The {PRODUCT} Operator uses the Infinispan Operator to deploy and manage the Infinispan infrastructure in a {PRODUCT} project. For optimal {PRODUCT} deployment on OpenShift, enable Inifinispan for your {PRODUCT} services. You can install the Infinispan infrastructure using the {PRODUCT} Operator UI in the OpenShift web console or using the {PRODUCT} CLI. This example uses the {PRODUCT} CLI to install Infinispan infrastructure and the {PRODUCT} Operator UI in the web console to verify that the infrastructure is enabled.
```

Comment:
Maybe it's worth to add Infinispan Operator docs: https://github.com/infinispan/infinispan-operator/blob/master/README.md

Reply:
Good call. Done.